src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -924,6 +925,17 @@ private function addAssetMapperSection(ArrayNodeDefinition $rootNode, callable $
                             ->info('The directory to store JavaScript vendors.')
                             ->defaultValue('%kernel.project_dir%/assets/vendor')
                         ->end()
+                        ->arrayNode('precompress')
+                            ->info('Precompress assets with Brotli, Zstandard and gzip.')
```

Comment:
shouldn't we be able to configure with compression formats we want, especially given that they require some dependencies to work (right now, if your server does not support the requirements for Zstandard, you cannot enable precompression at all instead of allowing to precompress for Brotli and gzip)

Reply:
Done. By default only supported formats will be enabled, and a warning will be displayed if a format cannot be logged because of a missing dependency.